cypress/e2e/spec.ts

```diff
@@ -102,27 +102,31 @@ describe('workspace-project App', () => {
 
     cy.intercept('GET', 'https://app.kendra.io/api/workflowCloud/listWorkflows', {
       fixture: 'listWorkflowsFlow.json'
-    });
+    }).as('listWorkflowsFlow.json');
 
     cy.intercept('GET', 'https://app.kendra.io/api', {
       fixture: 'flowList.json'
     }
     ).as('flowList.json');
 
     cy.visit('/workflowCloud/listWorkflows');
+    cy.contains('Submit').click();
```

Comment:
I made an issue for this change: https://github.com/kendraio/kendraio-app/issues/429
It might be a complex and tricky "can of worms".